datastore/leveldb/leveldbds.nim

```diff
@@ -0,0 +1,149 @@
+import std/options
+import std/tables
+import std/os
+import std/strformat
+
+import pkg/leveldbstatic
+import pkg/chronos
+import pkg/questionable
+import pkg/questionable/results
+import pkg/stew/byteutils
+from pkg/stew/results as stewResults import isErr
```

Comment:
Don't use `stew/result` use the standalone results package.

Reply:
Both fsds and sqliteds also use this same stew package. Do you want those changed as well?